app/controllers/support_interface/data_exports_controller.rb

```diff
@@ -8,6 +8,18 @@ def index
       @data_exports = DataExport.includes(:initiator).order(id: :desc).page(params[:page] || 1).per(30)
     end
 
+    def directory
+      @export_types = DataExport::EXPORT_TYPES
+    end
+
+    def view_export_information
+      @data_export = DataExport::EXPORT_TYPES[params[:data_export_type].to_sym]
+    end
+
+    def view_history
+      @data_exports = DataExport.includes(:initiator).where(name: params[:data_export_type].humanize).order(id: :desc).page(params[:page] || 1).per(30)
```

Comment:
It's not a massive deal, but the conversion that we do going back and forward using humanize or dasherize rather than just having a type attr feels a bit wrong.

It might be worth adding a column and backfilling wdyt?

Reply:
This would be done in a separate or though

Reply:
I agree with @davidgisbey — & this would be useful in other places too! (eg the TAD API where we currently look up relevant exports by human-readable name)

Reply:
@duncanjbrown & @davidgisbey I attempted the above - I don't think it's quite there but also hopefully not too far off from what you imaged? It'd be good to catch up with either of you today though and confirm this